Magefile.go

```diff
@@ -1,4 +1,5 @@
-//+build mage
+//go:build mage
+// +build mage
```

Comment:
what do these comments do?

Reply:
I think mage or vs code automatically changed those for me, but its the mage build tag so it knows what files to parse.

Reply:
these are go build tags (the first one is added automatically by the formatter)

> A magefile, instead of having a magic name (like Makefile), uses the go build tag //+build mage to indicate that mage should read it. Other than that, there’s nothing special about magefiles and you can name them whatever you like.